tracer/test/Datadog.Trace.TestHelpers/ProfilerHelper.cs

```diff
@@ -30,6 +36,61 @@ public class ProfilerHelper
             // clear all relevant environment variables to start with a clean slate
             EnvironmentHelper.ClearProfilerEnvironmentVariables();
 
+            // this is nasty, but it's the only way I could find to force
+            // a .NET Framework exe to run in 32 bit if required
+            if (EnvironmentTools.IsWindows()
+             && !EnvironmentHelper.IsCoreClr()
+             && !EnvironmentTools.IsTestTarget64BitProcess()
+             && Path.GetExtension(executable) == ".exe"
+             && !ExesToExcludeFromCorflags.Contains(Path.GetFileNameWithoutExtension(executable)))
+            {
+                string corFlagsExe = null;
```

Comment:
Consider moving the code inside this `if` out into its own method. Something like `SetCoreFlags(string assemblyPath, bool requires32Bit)`.

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/3632/commits/0501e7fcd11b8b059fc57ce41467d784f679cd79